### PR TITLE
Issue#923 Update VAT configuration page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- describes the relationship between the tax rate and the tax rule creation process.  The **Step 1** involves the confirmation of the needed tax class/rate in order to complete **Step 2**.
- fixes for #923 

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/tax/vat-configure.html

### CODE REVIEW

File: src/tax/vat-configure.md [hunk 1]
```diff
@@ -11,15 +11,29 @@ In certain business-to-business transactions, VAT is not assessed. Magento can v
...
 
 1. Ensure that there is a customer tax class that is appropriate to use with the VAT.
 
-   For this example, ensure that there is a customer tax class named Retail Customer. If Retail Customer does not exist, click <span class="btn">Add New Tax Rate</span> and add it.
+   For this example, ensure that there is a customer tax class named Retail Customer. If Retail Customer does not exist, click <span class="btn">Add New Tax Rate</span>.
 
     ![]({% link images/images/configuring-vat-tax-zones-and-rates.png %}){: .zoom}
     _Set up customer tax classes_
 
+1. Enter the **Tax Identifier** for the new tax class. All tax rates displayed in the **Tax Rate** field of the **Tax Rule Information**, for each tax rules.
```
Comment (leslietilling) [inline src/tax/vat-configure.md:25]:
```suggestion
1. Enter the **Tax Identifier** for the new tax class.

   All tax rates are displayed in the _Tax Rate_ field in the _Tax Rule Information_ when you create tax rules.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some edits for style and clarity

**Comment by ZhovnirMaria** [comment]
Thanks, @leslietilling 😊
Done✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**